applications/machine_learning/src/modules/Kconfig.ml_runner

```diff
@@ -14,7 +14,7 @@ if ML_APP_ML_RUNNER
 
 config ML_APP_ML_RUNNER_SENSOR_EVENT_DESCR
 	string "Description of handled sensor event"
-	default ""
+	default ML_APP_SENSOR_EVENT_DESCR
```

Comment:
I would suggest using this name also in `sensor_manager_def.h` file. Thanks to that, we would ensure consistency there too.